lib/topologies/read_preference.js

```diff
@@ -147,7 +144,8 @@ ReadPreference.prototype.slaveOk = function() {
 /**
  * Are the two read preference equal
  * @method
- * @return {boolean}
+ * @param {ReadPreference} readPreference the other read preference
```

Comment:
Instead of "the other read preference", I think "The read preference with which to check equality" would be more clear.